express-delete/server.js

```diff
@@ -0,0 +1,34 @@
+import express from 'express';
+
+const app = express();
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.get('/api/grades', (req, res, next) => {
+  res.json(Object.values(grades));
+  return next();
```

Comment:
Calling `next` after you send a response will allow any post-processing middleware to enhance the response. So that is good. On the other hand, we don't have any installed and won't install any during the LFZ course, so it's not necessary. It's up to you, of course.